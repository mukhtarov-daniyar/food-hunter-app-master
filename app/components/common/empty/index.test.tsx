import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Empty from './index';

it('renders correctly', () => {
  const tree = renderer.create(<Empty text={'text'} />).toJSON();
  expect(tree).toMatchSnapshot();
});
