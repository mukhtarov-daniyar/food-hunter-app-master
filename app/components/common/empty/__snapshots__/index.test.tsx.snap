// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "justifyContent": "center",
      "paddingHorizontal": 115.38461538461539,
    }
  }
>
  <Text
    allowFontScaling={false}
    style={
      [
        {
          "color": "#FF5252",
          "fontSize": 76.92307692307692,
        },
        undefined,
        {
          "fontFamily": "Ionicons",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        {},
      ]
    }
  >
    ï‹†
  </Text>
  <Text
    style={
      {
        "color": "#3D3D3D",
        "fontSize": 31.25,
        "marginTop": 19.23076923076923,
        "textAlign": "center",
      }
    }
  >
    text
  </Text>
</View>
`;
